lib/node_modules/@stdlib/stats/array/nanmskmin/test/test.js

```diff
@@ -0,0 +1,327 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var toAccessorArray = require( '@stdlib/array/base/to-accessor-array' );
+var isNegativeZero = require( '@stdlib/math/base/assert/is-negative-zero' );
+var BooleanArray = require( '@stdlib/array/bool' );
+var Complex128Array = require( '@stdlib/array/complex128' );
+var nanmskmin = require( './../lib/main.js' );
```

Comment:
```suggestion
var nanmskmin = require( './../lib' );
```

We shouldn't be bypassing the package entry point in `stats/array` packages.